doc/how_to/notebook/other_nb.md

```diff
@@ -8,6 +8,12 @@ This guide addresses how to develop apps in Google Colab, VSCode, nteract, and o
 
 In the Google Colaboratory notebook, first make sure to load the `pn.extension()`. Panel objects will then render themselves if they are the last item in a notebook cell. Please note that in Colab rendering for each notebook cell is isolated, which means that every cell must reload the Panel extension code separately. This will result in somewhat slower and larger notebook than with other notebook technologies.
 
+* Be aware: using Panel with ipywidgets in Colab does not work, due to Colab's architecture constraints.
```

Comment:
I would suggest turning this into a nice warning admonition and making the text more specific.

````markdown

:::{warning}
Panels [Ipywidget](../../reference/panes/IPyWidget.ipynb) pane does not work in Colab due to architecture constraints.
:::

````markdown

Reply:
Agree but I'm not sure what the optimal text should be, in terms of warning (how are those usually formulated on the site?) and technical specifics.Does the Discord discussion I referred to help?

Reply:
Iteration: * Be aware: using ipywidgets with Panel in Colab does currentlly not work, due to Colab's architecture constraints.